### PR TITLE
Feature/movingpermit

### PR DESCRIPTION
**Pull request by thiteixeira**
This intent has been tested on my end.

Need to merge the following files for it to work:
```
mycity/mycity/intents/moving_permit.py
mycity/mycity/logger.py
mycity/mycity/mycity_controller.py
mycity/mycity/utilities/gis_utils.py
```

Also, make sure the timeout on lambda function `Basic settings` is set higher than 3 seconds (mine is 1 minute).

This PR closes issue #87

### CODE REVIEW

**Comment by curranjm** [comment]
Rebased this branch onto project master to get rid of changes not related to the moving permit intent.

File: mycity/mycity/utilities/gis_utils.py [hunk 1]
```diff
@@ -7,8 +7,12 @@
 """
 
 from arcgis.features import FeatureLayer
-
 import mycity.utilities.google_maps_utils as g_maps_utils
+from arcgis.gis import *
```
Comment (curranjm) [inline mycity/mycity/utilities/gis_utils.py]: Can we just import what we need from the library, or do `import arcgis.gis as gis` and access the members we need from that?

File: mycity/mycity/utilities/gis_utils.py [hunk 2]
```diff
@@ -7,8 +7,12 @@
...
+from arcgis.geocoding import geocode, reverse_geocode
+from math import sin, cos, sqrt, atan2, radians
+
+dev_gis = GIS()  # this is needed to use geocoding
```
Comment (curranjm) [inline mycity/mycity/utilities/gis_utils.py]: Do we know exactly what this is doing? Do we need a login to use this in production?

**Comment by thiteixeira** [comment]
Will defer this for now

**Pull request closed without merging**